homeassistant/components/sensor/usps.py

```diff
@@ -120,8 +122,10 @@ def state(self):
 
     def update(self):
         """Update device state."""
+        import datetime
         import myusps
-        self._state = len(myusps.get_mail(self._session))
```

Comment:
Good catch! Mutable default arguments bites us here! I guess this is actually a bug in the `myusps` lib.